homework/jenya_s/homework7.py

```diff
@@ -0,0 +1,14 @@
+def magic_range():
+
+    min_lim, max_lim = None, None
+
+    for i in range(-200, 1000):
+        if min_lim == None and i is i and i+1 is i+1:
```

Comment:
when you compare some object with None you should use `is` operator.

Reply:
done